canopen/node/remote.py

```diff
@@ -154,4 +154,4 @@ def load_configuration(self):
                         self.__load_configuration_helper(subobj.index, subobj.subindex, subobj.name, subobj.value)
             elif isinstance(obj, ODVariable) and obj.writable and (obj.value is not None):
                 self.__load_configuration_helper(obj.index, None, obj.name, obj.value)
-        self.pdo.read()  # reads the new configuration from the driver
+        self.pdo.read(from_od=from_od)  # reads the new configuration from the driver
```

Comment:
This seems to be the reason for adding the `from_od` argument. The current imeplementations calls `self.pdo.read()` with default parameters will read from the remote.

Maybe it's better to do `self.pdo.read(True)` and not add the `from_od` argument? The function description does not allude to it doing something against the remote.